.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "nfcore",
+    "image": "nfcore/gitpod:latest",
+    "postCreateCommand": "python -m pip install --upgrade -r ../requirements-dev.txt -e ../",
+    "remoteUser": "gitpod",
+
+    // Configure tool-specific properties.
+    "customizations": {
+        // Configure properties specific to VS Code.
+        "vscode": {
+            // Set *default* container specific settings.json values on container create.
+            "settings": {
+                "python.defaultInterpreterPath": "/opt/conda/bin/python",
+                "python.linting.enabled": true,
+                "python.linting.pylintEnabled": true,
+                "python.formatting.autopep8Path": "/opt/conda/bin/autopep8",
+                "python.formatting.yapfPath": "/opt/conda/bin/yapf",
+                "python.linting.flake8Path": "/opt/conda/bin/flake8",
+                "python.linting.pycodestylePath": "/opt/conda/bin/pycodestyle",
+                "python.linting.pydocstylePath": "/opt/conda/bin/pydocstyle",
+                "python.linting.pylintPath": "/opt/conda/bin/pylint"
+            },
+
+            // Add the IDs of extensions you want installed when the container is created.
+            "extensions": [
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "nf-core.nf-core-extensionpack",
+                "codezombiech.gitignore",
+                "esbenp.prettier-vscode",
+                "EditorConfig.EditorConfig",
+                "Gruntfuggly.todo-tree",
+                "mechatroner.rainbow-csv",
+                "derwat.indent-rainbow",
+                "streetsidesoftware.code-spell-checker"
```

Comment:
These are all part of the `nf-core-extensionpack`, right? Do they need to be listed here? I assumed that just installing `nf-core.nf-core-extensionpack` would handle the rest of these..

Reply:
Correct. Removed